HelpSource/Tutorials/Mark_Polishook_tutorial/05_The_network.schelp

```diff
@@ -25,7 +25,7 @@ s = Server("aServer", NetAddr("127.0.0.1", 56789));		// "localhost" is a synonym
 s.boot;
 
 // define a synthesis engine
-SynthDef("sine", { Out.ar(0, SinOsc.ar(440, 0, 0.2)) }).send(s);
+SynthDef("sine", { Out.ar(0, SinOsc.ar(440, 0, 0.2)) }).add;
```

Comment:
keep send, it is required (I think) because of the remote server. At least it is better to keep it. Please also apply the same to the other cases in this file and its translations to Japanese.

Reply:
Thank you, and my apologies for the change. I have reverted it in https://github.com/supercollider/supercollider/pull/7117/commits/f8d8f2c470aef9d1cd15d9e57db25f0122d98341.

Reply:
No, it is still the same. Please use send, thanks!

Reply:
Thanks and sorry! 
Eone in https://github.com/supercollider/supercollider/pull/7117/commits/b5aaedbf45416cdc3221a55938cddc63aa5887fa